userprofile/static/userprofile/copy_token.js

```diff
@@ -0,0 +1,21 @@
+function copyToken(text, el) {
+  var elOriginalText = el.attr("data-original-title");
+
+  $("#api-access-token").select();
+  try {
+    const succeed = document.execCommand("copy");
+    const msg = succeed ? "Copied!" : "Not copied!";
```

Comment:
A+ has a function for translating strings in JavaScript: `_("Copied!")`

The translations for JS code are defined in these JSON files:
https://github.com/apluslms/a-plus/tree/master/assets/js-translations